plugin-core/src/main/java/appland/toolwindow/signInView/SignInViewPanel.java

```diff
@@ -84,7 +84,7 @@ public boolean onBeforeBrowse(CefBrowser browser, CefFrame frame, CefRequest req
 
         htmlPanel.getJBCefClient().addDisplayHandler(new ConsoleInitMessageHandler(this::initWebView), htmlPanel.getCefBrowser());
 
-        htmlPanel.loadURL(AppMapWebview.SignIn.getIndexHtmlUrl());
+        htmlPanel.loadURL(AppMapWebview.getBaseUrlWithPath("signin.html"));
```

Comment:
What's the reason to avoid AppMapWebview.SignIn? That would keep all HTML file paths in the same place.